mozilla-release/mobile/android/base/locales/en-US/android_strings.dtd

```diff
@@ -470,7 +470,7 @@
 <!-- Cliqz start -->
 <!-- change text of normal and private tabs -->
 <!ENTITY tabs_normal "TABS">
-<!ENTITY tabs_private "FORGET TABS">
+<!ENTITY tabs_private "GHOST TABS">
```

Comment:
if we rename it for Ghostry only, you should do it in branding file
cliqz-android/mozilla-release/mobile/android/branding/ghostery/locales/en-US/brand.dtd
cliqz-android/mozilla-release/mobile/android/branding/ghostery-alpha/locales/en-US/brand.dtd
cliqz-android/mozilla-release/mobile/android/branding/cliqz/locales/en-US/brand.dtd
cliqz-android/mozilla-release/mobile/android/branding/cliqz-alpha/locales/en-US/brand.dtd